docs/guides/simbrief.md

```diff
@@ -11,4 +11,49 @@ This guide is also available in [video form](https://youtu.be/cSY-r83pXio) on ou
 
 :::info
 This guide assumes you already have a Navigraph account and know how to set up a SimBrief flight plan. If you do not, our [video](https://youtu.be/cSY-r83pXio) covers this in detail.
-:::
\ No newline at end of file
+:::
+
+## Finding Your User ID
+Before you can start importing your SimBrief flight plans into the aircraft, you must first know what your Simbrief User ID is. This allows the aircraft, and your flightplan, to seemlessly synchronise with each other. 
```

Comment:
Maybe change the last sentence to say something like "allows the aircraft to fetch the flight plan from SimBrief". User ID -> Pilot ID, capitalization on SimBrief.